### PR TITLE
Add VS Code Insiders Flatpak

### PR DESCRIPTION
**Pull request by fooishbar**
Add a separate Flatpak for the Insiders stream of VS Code.

This was just done by copying the non-Insiders files and changing what was required. Maybe there's a better opportunity for using sed or environment variables or something? Not sure.

### CODE REVIEW

File: com.visualstudio.code.insiders.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <screenshot type="default">
+      <image>https://code.visualstudio.com/home/home-screenshot-linux-lg.png</image>
+      <caption>Editing TypeScript and searching for extensions</caption>
+    </screenshot>
+  </screenshots>
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-info">moderate</content_attribute>
+  </content_rating>
+  <releases>
+    <release version="1.41.1" date="2019-12-19"/>
```
Comment (fooishbar) [inline com.visualstudio.code.insiders.appdata.xml]: Oops.

**Comment by fooishbar** [comment]
The icons were generated by taking the provided SVG and manually downscaling in GIMP. I have no idea if this represents the state of the art.

**Comment by fooishbar** [comment]
I've rebased this on top of actually-current master (oops), added a version entry for appstream, added some disclaimer text noting that the Insiders build is not necessarily suitable for general usage, and retargeted at the beta branch.

Unfortunately the build now fails because we have a different app ID.

**Comment by nedrichards** [comment]
And to be clear, the different app-id is because this is designed to run side by side with the main version as a 'dev channel' version. As such it would also benefit from the automatic PR and rebase infra we have on flathub.

**Comment by eszlari** [comment]
For the beta branch you don't need to change the APP_ID, see e.g.:
https://github.com/flathub/org.shotcut.Shotcut/tree/beta

The question is, will you maintain this? That is, merge PRs opened by the bot? How many can we expect?

**Comment by nedrichards** [comment]
OK, in order to keep the separate app-id we'll need this to be submitted as a new app. I'll make sure it only has the beta channel on that app. So follow the guidelines here please https://github.com/flathub/flathub/wiki/App-Submission

**Comment by fooishbar** [comment]
@eszlari Updates come every couple/few days, by the looks. It could be maintained by the existing VSC maintainers, or I'm happy to help with Insiders-specific issues.

**Comment by fooishbar** [comment]
Retargeted with a new PR as flathub#1422.

**Comment by eszlari** [comment]
> It could be maintained by the existing VSC maintainers

Who?

**Pull request closed without merging**